iib/workers/tasks/build.py

```diff
@@ -116,45 +116,36 @@ def _create_and_push_manifest_list(request_id, arches):
     :rtype: str
     :raises IIBError: if creating or pushing the manifest list fails
     """
+    buildah_manifest_cmd = ['buildah', 'manifest']
     output_pull_spec = get_rebuilt_image_pull_spec(request_id)
-    log.info('Creating the manifest list %s', output_pull_spec)
+    log.info('Creating the manifest list %s locally', output_pull_spec)
+    sorted_arches = sorted(arches)
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
-        manifest_yaml = os.path.abspath(os.path.join(temp_dir, 'manifest.yaml'))
-        with open(manifest_yaml, 'w+') as manifest_yaml_f:
-            manifest_yaml_f.write(
-                textwrap.dedent(
-                    f'''\
-                    image: {output_pull_spec}
-                    manifests:
-                    '''
-                )
+        for arch in sorted_arches:
```

Comment:
would it be possible to use line 122 here? if you don't use sorted_aches anywhere else?

Reply:
But I am using it on Line 130